.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.1</version>
+    <version>1.3</version>
```

Comment:
@Kevin-CB See https://github.com/jenkinsci/incrementals-tools/releases for the release notes for this component. This component is used to compute the version numbers for incrementals and CD releases, see https://www.jenkins.io/doc/developer/plugin-development/incrementals/ and https://www.jenkins.io/doc/developer/publishing/releasing-cd/.